velox/functions/prestosql/window/FirstLastValue.cpp

```diff
@@ -34,10 +34,16 @@ class FirstLastValueFunction : public exec::WindowFunction {
   explicit FirstLastValueFunction(
       const std::vector<exec::WindowFunctionArg>& args,
       const TypePtr& resultType,
+      bool ignoreNulls,
       velox::memory::MemoryPool* pool)
       : WindowFunction(resultType, pool, nullptr) {
     VELOX_CHECK_NULL(args[0].constantValue);
-    argIndex_ = args[0].index.value();
+    valueIndex_ = args[0].index.value();
+
+    ignoreNulls_ = ignoreNulls;
+    if (ignoreNulls_) {
+      nullsVector_ = BaseVector::create<FlatVector<bool>>(BOOLEAN(), 0, pool_);
```

Comment:
Do we need a vector or just a buffer would be enough?

Reply:
I was trying to avoid reallocating the buffer, but have now used buffer instead.